assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -108,4 +109,12 @@ export function registerDefaults( widgetsAPI ) {
 		style: WIDGET_AREA_STYLES.BOXES,
 		priority: 4,
 	}, CONTEXT_PAGE_DASHBOARD );
+
+	widgetsAPI.registerWidget( AREA_PAGE_DASHBOARD_POPULARITY, {
+		title: __( 'Search for individual page or post information', 'google-site-kit' ),
+		style: WIDGET_AREA_STYLES.BOXES,
+		priority: 100,
+		width: WIDGET_WIDTHS.HALF,
+		component: URLSearchWidget,
```

Comment:
This should pass `wrapWidget: false`, otherwise the `Widget` manually rendered by the component will be wrapped in another `Widget`.